skafka/src/main/scala/com/evolutiongaming/skafka/consumer/ConsumerConfig.scala

```diff
@@ -33,15 +33,19 @@ final case class ConsumerConfig(
 ) {
 
   def bindings: Map[String, String] = {
-    val groupIdMap = groupId.fold(Map.empty[String, String]) { groupId => Map((C.GROUP_ID_CONFIG, groupId)) }
+    val groupIdMap = groupId.fold(Map.empty[String, String]) { groupId =>
```

Comment:
I think it's just better to make `autoCommitInterval` as Option in backward compatible way

Reply:
It will still allow setting `autoCommitInterval` without `groupId` won't it?

Reply:
yes it will, and it will warn - exact same behaviour as original Kafka client

Reply:
ok, I see your point

Reply:
made `autoCommitInterval` optional, please review